auacm/app/modules/submission_manager/judge.py

```diff
@@ -33,6 +34,9 @@
     'cpp': 1,
     'go': 1
 }
+DB_STATUS = ['', 'compile', 'start', 'runtime', 'timeout', 'wrong', 'good']
+EVENT_STATUS = ['', 'compile', 'running', 'runtime', 'timeout', 'incorrect',
+                'correct']
```

Comment:
Yeah, we should be able to do that. I'm going to prefer no changes on the DB though, so we'll have to make a few changes on the front-end. It'll be a lot nicer to see things capitalized anyway.